.github/actions/spell-check/expect.txt

```diff
@@ -15,6 +15,7 @@ accessibilityinsights
 Acl
 aclapi
 AColumn
+Accesible
```

Comment:
this should have double S (unless it's Spanish)

Reply:
Fixed!